lib/ruby-mws/api/base.rb

```diff
@@ -7,7 +7,7 @@ module API
     class Base
       include HTTParty
       parser MWS::API::BinaryParser
-      debug_output $stderr  # only in development
+      debug_output $stderr if Rails.env.development? # only in development
```

Comment:
You're assuming here that it will only be used within a rails app. Might be better to pass in the `development` status in the MWS initialize call.